ultravox/inference/infer.py

```diff
@@ -33,22 +33,33 @@ def __init__(
     def infer(
         self,
         sample: datasets.VoiceSample,
-        max_tokens: Optional[int] = None,
+        max_new_tokens: Optional[int] = None,
```

Comment:
I think we want to keep the same parameter names as in the base class.

Reply:
Yes I think it should change in base class and all the children.
Python never complains, so I guess keeping the interface consistent is on the developer :)

Reply:
I changed it back to `max_tokens` to be consistent with the base class (to be consistent with OPENAI parameter name)